src/styles/landing-page.less

```diff
@@ -1,88 +1,34 @@
-@sidebar-width: 320px;
-
-.landing-page {
-  background-color: @color-pf-black-200;
-}
-
-.landing-page-container {
-  background-color: @color-pf-black-200;
-  bottom: 0;
-  display: flex;
-  flex-direction: column;
-  height: 100%;
-  min-height: 100%;
-  overflow: visible;
-  position: relative;
-
-  @media (min-width: @screen-sm-min) {
-    top: @navbar-pf-height;
-  }
-
-  &.snap-up-shown {
-    padding-bottom: 40px;
-  }
-}
-
 .landing-main-area {
-  margin: 15px 20px;
-  padding-right: 10px;
-
-  // TODO: Remove corresponding style in services-view.less (card-expansion)
-  // when sidebar is no longer fixed position.
-  @media (min-width: @screen-sm-min) {
-    margin-right: @sidebar-width;
+  padding: (@grid-gutter-width / 2); // this is temp
```

Comment:
@cshinn, you're gonna want to remove this to get the styling of the search and apps area right, but I ended up putting it in to make the page look ok in the interim.